pkg/cloudprovider/providers/azure/azure_wrap.go

```diff
@@ -54,25 +56,72 @@ func ignoreStatusNotFoundFromError(err error) error {
 	return err
 }
 
+// cache used by getVirtualMachine
+var vmCache = cache.New(15*time.Second, 10*time.Minute)
```

Comment:
what's `10*time.Minute` for? add comments for expiration time explaination

Reply:
Updated